app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -197,12 +195,12 @@ export default {
       'form.userType',
       'form.versions',
     ]),
+    NOTIFICATIONS_TYPES() {
```

Comment:
Alphabetical ordering of computed attribute elements - should be after ID_MODE

Reply:
moved to proper location